src/routes/+page.svelte

```diff
@@ -124,15 +124,16 @@
 	</div>
 {/if}
 
-{#snippet skipLine(text: string, plus1 = false)}
+{#snippet skipLine(text: string, status: string = 'cleared')}
```

Comment:
status should be `Level['status']` and not accept a default

Reply:
Is there a default that should be passed to the starting skipLine?

```svelte
{@render skipLine(`Start, ${initial_skips.value} ${plural('skip', initial_skips.value)}`
```

Reply:
in_progress I guess

Reply:
Actually I think skipLine should be passed a deltaSkips number instead of the status, that makes more sense